heuristics/__init__.py

```diff
@@ -3,13 +3,27 @@
 # import heuristic implementations here
 from .count import event_count, message_count
 from .ratio import average_message_length, message_to_command_ratio
+from .feng import average_time_between_message_and_command, ratio_of_commands_without_message
```

Comment:
Seems like this file is missing - did you forget to `git add` it before committing?